changelogs/1.23.3.yaml

```diff
@@ -1,2 +1 @@
 date: December 2, 2022
-
```

Comment:
cc @phlax 
Seems like the automated tool generated a file with  empty line which conflicted with format checks

Reply:
its possible - we added the linter fairly recently - i reckon we can just fix the file for now - ill raise a ticket ...

https://github.com/envoyproxy/envoy/issues/24918